e2e/test/scenarios/organization/bookmarks-question.cy.spec.js

```diff
@@ -47,6 +47,7 @@ describe("scenarios > question > bookmarks", () => {
     cy.log("Turn the model back into a question");
     openQuestionActions();
     cy.findByRole("dialog").contains("Turn back to saved question").click();
+    cy.wait(1000);
```

Comment:
Had to add this delay for the test to work

Reply:
it's not a way to go in 99% of cases

Reply:
Found a better fix indeed!